packages/paste-website/src/pages/components/input/index.mdx

```diff
@@ -239,12 +239,24 @@ Help Text should also include any important contextual information about what th
 
 <LivePreview scope={{Input, Label, HelpText}} language="jsx">
   {`<>
-  <Label htmlFor="reload_amount">Reload amount</Label>
+  <Label htmlFor="reload_amount">Number Input</Label>
   <Input aria-describedby="reload_amount_help" id="reload_amount" name="reload_amount" defaultValue="0" type="number" min="0" max="1000" />
   <HelpText id="reload_amount_help">Maximum amount is $1,000.</HelpText>
 </>`}
 </LivePreview>
 
+#### Internationalization
+
+To internationalize Number Input, provide translated strings to be used for the increment and decrement button accessible labels. Use the `i18nStepUpLabel` and `i18nStepDownLabel` props.
```

Comment:
```suggestion
Provide translated strings for the 'increment' and 'decrement' button labels to make NumberInput localized. The `i18nStepUpLabel` and `i18nStepDownLabel` props should be used for this purpose.
```

Reply:
This was marked as resolved without any changes afaict. It reads a little weird to me, but not gonna block on it.

Reply:
weird, I made the changes but they got lost